sequelizeDatabase/sequelFunctions.js

```diff
@@ -259,23 +259,15 @@ deleteIndividual = IndividualID => {
   //
   // Modfying this to avoid a race condition -- Aaron
   return new Promise((resolve, reject) => {
-    models.EmployeeHistory.destroy({
-      where: {
-        IndividualID: IndividualID
-      }
-    })
-      .then(deletedHistory => {
-        models.Individuals.destroy({
-          where: {
-            IndividualID: IndividualID
-          }
-        }).then(deletedIndividual => {
-          console.log("deletion resolved");
-          resolve(deletedIndividual);
-        });
-      })
-      .catch(err => {
-        console.error(err);
+      models.Individuals.destroy({
```

Comment:
I know that this is very late since most functions are already written this way, but instead of returning promise and using .then you can use async functions and awaits since they are much more readable. Please keep this since it works, and maybe after the MVP we can refactor it together.

 So for this case the function would be:
```javascript
const deletedIndividual = await models.Individuals.destroy({where: {IndividualID: IndividualID}});
console.log("New deletion of an individual.");
return deletedIndividual;  //Notice that is it a return and not a resolve
```